backup/dependencies.go

```diff
@@ -278,12 +278,13 @@ func GetDependencies(connectionPool *dbconn.DBConn, backupSet map[UniqueID]bool,
 	query := `SELECT
 	coalesce(id1.refclassid, d.classid) AS classid,
 	coalesce(id1.refobjid, d.objid) AS objid,
-	coalesce(id2.refclassid, d.refclassid) AS refclassid,
-	coalesce(id2.refobjid, d.refobjid) AS refobjid
+	coalesce(id3.refclassid, coalesce(id2.refclassid, d.refclassid)) AS refclassid,
+	coalesce(id3.refobjid, coalesce(id2.refobjid, d.refobjid)) AS refobjid
```

Comment:
we can write the same more compact way:
```suggestion
	coalesce(id3.refclassid, id2.refclassid, d.refclassid) AS refclassid,
	coalesce(id3.refobjid, id2.refobjid, d.refobjid) AS refobjid
```